tests/test_pytypes.py

```diff
@@ -604,6 +604,7 @@ def test_number_protocol():
 def test_list_slicing():
     li = list(range(100))
     assert li[::2] == m.test_list_slicing(li)
```

Comment:
```suggestion
    assert li[0:-1:2] == m.test_list_slicing(li)
```

This might be the easiest fix for the discrepancy here.